hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java

```diff
@@ -1760,7 +1760,7 @@ UserGroupInformation getTicket() {
       return ticket;
     }
     
-    private int getRpcTimeout() {
```

Comment:
Can we do the HADOOP changes in a separate PR with a HADOOP JIRA?
Something like "Allow socket factory for connections"

Reply:
copy, I will do it.

Reply:
Actually that's HADOOP-13144.
Can you do a PR there? I think we are safe merging that.